src/PrizeVault.sol

```diff
@@ -256,6 +281,13 @@ contract PrizeVault is TwabERC20, Claimable, IERC4626, ILiquidationSource, Ownab
         return _underlyingDecimals;
     }
 
+    /// @inheritdoc IERC20
+    /// @dev Adds the yield fee balance to the total supply since it's cheaper to keep track of those shares
+    ///      internally instead of doing an additional mint on every liquidation.
+    function totalSupply() public view override(TwabERC20, IERC20) returns (uint256) {
+        return twabController.totalSupply(address(this)) + yieldFeeBalance;
```

Comment:
The yieldFeeBalance doesn't belong here; this breaks the ERC20 spec!  This means the accumulated Transfer events won't sum correctly to the total supply.

This balance should be separate from the ERC20 total supply.

Reply:
Fixed here: https://github.com/GenerationSoftware/pt-v5-vault/pull/75/commits/ae15c293105a5a4bfe1caec57be7d8052291e7a6

Added a `totalDebt` function that accounts for *all* assets owed (shares and yield fee accrual) to keep the accounting clean and concise.